### DISCUSSION TITLE
Cartoonizer

### DISCUSSION THREAD

**Discussion by tbc725** [codeproject/CodeProject.AI-Server#194] [Q&A] [answered]
Question, within the Cartoonizer module, there are 4 default models. Is it possible to add other models from HuggingFace to this module? I assume the models would need to be compatible, but I’d appreciate any guidance on how to approach this.

**Comment by ChrisMaunder** [comment, marked as answer]
I'm sure it's possible, but not planned. The models will have to be compatible with the code. If anyone wants to clone this repo and add support then that would be awesome

**Reply by tbc725** [reply]
Ok. Thank you for the data on this.
